jupyterlab/docker-compose.yml

```
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+
+  movies-jupyterlab:
+    build: ./image/
+    image: movies-jupyterlab:1.0.0
+    container_name: movies-jupyterlab
+    restart: unless-stopped
+    ports:
+      - 8888:8888
+    volumes:
+      - ./volumes/ipynb/:/home/jovyan/work/:rw
```

@kerphi @yannnic  Je ne suis pas favorable à ce qu'on laisse aux utilisateurs la possibilité sauvegarder le notebook, car il y a trop de risques qu'un utilisateur le casse accidentellement, ce qui le rendrait inutilisable pour les autres et modifierait nos résultats de recherche.

Je propose qu'on supprime l'accès en écriture, ce qui permettra aux utilisateurs d'expérimenter (changer les requêtes, voir les résultats, etc.) mais pas de sauvegarder les résultats.

```suggestion
      - ./volumes/ipynb/:/home/jovyan/work/:r
```

---

Oui bonne idée la lecture seule, ca permettrait de définir par où arrive l'information !
J'ai aussi repéré ceci https://github.com/jupyterlab/jupyterlab-git qui pourrait être une piste pour mieux gérer les fichiers ipynb ?